__tests__/app.test.js

```diff
@@ -43,27 +43,28 @@ return request(app)
 
 
 describe('GET /api/articles/:article_id', () => {
-  test('200: responds with the correct article object for a valid ID', () => {
+  test('200: responds with the correct article object including comment_count', () => {
```

Comment:
Rather than refactoring an old test here, we should be writing a new one to show the comment_count is there 👍 If you find that the older test starts to fail though, you may need to change it so it uses a more flexible matcher 🙂